eslintrc.json

```diff
@@ -37,19 +39,10 @@
 
   },
   "rules": {
-    "array-bracket-spacing": [2],
```

Comment:
Is there a way you could explain why some rules have been removed while some stayed?

Reply:
All styling rules were removed as they are handled by prettier, The rest of the rules, which are more about imports, validation, proper practices etc are kept, as these are not handled by prettier